lib/broccoli/broccoli-typescript.js

```diff
@@ -242,12 +243,31 @@ class BroccoliTypeScriptCompiler extends Plugin {
    * This issue is fixed in https://github.com/Microsoft/TypeScript/pull/5620.
    * Once we switch to TypeScript 1.8, we can remove this method.
```

Comment:
We still need to fix sources. I need to spend much more time refactoring this plugin and the rest.